pkg/k8s/types.go

```diff
@@ -0,0 +1,10 @@
+package k8s
+
+const (
+	// DestinationRuleCrdName defines the CRD name for DestinationRule as in Istio.
+	DestinationRuleCrdName string = "destinationrules.networking.istio.io"
+	// DestinationRuleKind defines the Kind name for DestinationRule as in Istio.
+	DestinationRuleKind string = "DestinationRule"
+	// DestinationRuleAPIVersion defines the API version for DestinationRule as in Istio.
+	DestinationRuleAPIVersion string = "networking.istio.io/v1alpha3"
```

Comment:
Does it make sense to read these values from ENV config?

Reply:
I think, its not gonna be updated too frequently. We can keep it hardcoded here for now.